esmvaltool/_data_finder.py

```diff
@@ -233,22 +233,27 @@ def get_input_fx_dirname_template(variable, rootpath, drs):
         if isinstance(input_dir, six.string_types):
             dir2 = replace_tags(input_dir, variable, i=fx_ind)
         elif _drs in input_dir:
-            dir2 = replace_tags(input_dir[_drs], variable, i=fx_ind)
+            try:
+                insts = cmip5_dataset2inst(variable['dataset'])
+            except KeyError as msg:
+                logger.debug('CMIP5 dataset2inst: %s', msg)
+                insts = 0
+            dirs2 = []
+            if isinstance(insts, list):
+                for j in range(len(insts)):
+                    dir2 = replace_tags(
+                        input_dir[_drs], variable, j=j, i=fx_ind)
+                    dirs2.append(dir2)
+            else:
+                dir2 = replace_tags(
+                    input_dir[_drs], variable, j=None, i=fx_ind)
+                dirs2.append(dir2)
         else:
             raise KeyError(
                 'drs {} for {} project not specified in config-developer file'
                 .format(_drs, project))
 
-        # Replace seaIce realm by ocean realm
-        path_elements = dir2.split(os.path.sep)
-        if "seaIce" in path_elements:
-            old_dir = dir2
-            dir2 = dir2.replace("seaIce", "ocean")
-            logger.info(
-                "Replaced path to fx files %s by %s for seaIce"
-                "diagnostics", old_dir, dir2)
-
-        dirname_template = os.path.join(dir1, dir2)
+        dirname_template = [os.path.join(dir1, dir_2) for dir_2 in dirs2]
         dirs.append(dirname_template)
```

Comment:
It should be `extend`, with `append` you create a list of lists.

Reply:
sorry, I just forgot to remove the list of lists making, my bad :)